test/scraper/meck_pomm_landtag_scraper_detail_test.rb

```diff
@@ -53,6 +53,30 @@ def setup
       }, paper)
   end
 
+  test 'extract details 6/4640 without minisitry' do
+    html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/meck_pomm_landtag_scraper_detail_6_4640.html')))
+    body = html.search("//table[@id = 'parldokresult']")
+    paper = @scraper.extract(body.at_css('.title'))
+
+    assert_equal(
+        {
+            legislative_term: '6',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.